build.gradle

```diff
@@ -82,11 +82,11 @@ allprojects {
 version '5.0.0-beta2-SNAPSHOT'
 group 'com.braintreepayments'
 ext {
-    compileSdkVersion = 34
+    compileSdkVersion = 35
```

Comment:
Do we need to call this out in the changelog at all?

Reply:
Oh yeah def I zoned out on the test portion. I'll add it in.